### PR TITLE
add IA2_ROLE_CONTENT_DELETION/INSERTION roles

### PR DESCRIPTION
**Pull request by asurkov**

### CODE REVIEW

File: api/AccessibleRole.idl [hunk 1]
```diff
@@ -302,5 +302,15 @@ enum IA2Role {
    * A bar that serves as a level indicator to, for instance, show
    * the strength of a password or the charge of a battery.
    */
-  IA2_ROLE_LEVEL_BAR
+  IA2_ROLE_LEVEL_BAR,
+
+  /** Content previously deleted or proposed to be deleted, e.g. in revision
```
Comment (jcsteh) [inline api/AccessibleRole.idl]: nit: "proposed to be deleted" -> "proposed for deletion"

File: api/AccessibleRole.idl [hunk 2]
```diff
@@ -302,5 +302,15 @@ enum IA2Role {
...
+   history or a content view providing suggestions from reviewers.
+  */
+  IA2_ROLE_CONTENT_DELETION,
+
+  /** Content previously inserted or proposed to be inserted, e.g. in revision
```
Comment (jcsteh) [inline api/AccessibleRole.idl]: nit: "proposed to be inserted" -> "proposed for insertion"

File: api/AccessibleRole.idl (hunk 1 above)
Comment (aleventhal) [reply api/AccessibleRole.idl]: Also I messed up the comment format a little. Each line needs a *

Comment (aleventhal) [reply api/AccessibleRole.idl]: And there are some other mistakes in the comment format. I sent you a new block of text.

**Comment by asurkov** [comment]
comment style is messy, I can see at least 3 types of different styles, but
your one seems the most popular one

**Pull request merged**